### DISCUSSION TITLE
Error setting up Selfhosted LiveSync on replica Obsidian (using IBM Cloudant)

### DISCUSSION THREAD

**Discussion by ehammett** [vrtmrz/obsidian-livesync#319] [Q&A]
Not sure if I am doing something wrong, but I have my initial Obsidian (MacOS) setup with the database using IBM Cloudant, working fine. However, when I try to copy the URI on rmy second Obsidian (Linux AppImage) installation I get the error "Setup URI looks wrong". Any suggestions?

This is the format of my URI which works fine on the first setup:
https://xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx-bluemix.cloudantnosqldb.appdomain.cloud

**Comment by vrtmrz** [comment]
Sorry for being late! And for ambiguous terms.
Setup URI is not the URL of the remote database. Setup URI could be made by `Copy the setup URI` from the command palette of Obsidian that already had been configured. This should be `obsidian://setuplivesync?settings=%5B%22VGET9UPjSG3...`
